vividus-plugin-applitools/build.gradle

```diff
@@ -1,5 +1,7 @@
 project.description = 'VIVIDUS plugin for Applitools'
 
+ext.eyesVersion = '5.50.0'
```

Comment:
Is dependabot able to process this?

Reply:
https://github.com/vividus-framework/vividus/commit/d0139605d22dc3dac80bfb297eeb4d81129d9c6d